ocs_ci/deployment/hosted_cluster.py

```diff
@@ -0,0 +1,1127 @@
+        self.create_storage_class_claim_cephfs()
+        logger.info("Creating storage class claim rbd")
+        self.create_storage_class_claim_rbd()
+        logger.info("Verify Storage Class cephfs exists")
+        if not self.storage_class_exists(constants.CEPHFILESYSTEM_SC):
+            logger.error("Storage Class cephfs does not exist")
+            return False
+        logger.info("Verify Storage Class rbd exists")
+        if not self.storage_class_exists(constants.CEPHBLOCKPOOL_SC):
+            logger.error("Storage Class rbd does not exist")
+            return False
+        return True
+
+    @kubeconfig_exists_decorator
+    def odf_client_installed(self):
+        """
+        Check if ODF client is installed
+
+        Returns:
+            bool: True if ODF client is installed, False otherwise
+        """
+        logger.info("Waiting for ODF client CSV's to be installed")
+        timeout_wait_csvs = 10
+        timeout_wait_pod = 5
+
+        try:
+            sample = TimeoutSampler(
+                timeout=timeout_wait_csvs * 60,
+                sleep=15,
+                func=check_all_csvs_are_succeeded,
+                namespace=self.namespace_client,
+                cluster_kubeconfig=self.cluster_kubeconfig,
+            )
+            sample.wait_for_func_value(value=True)
+
+            app_selectors_to_resource_count_list = [
+                {"app.kubernetes.io/name=ocs-client-operator-console": 1},
+                {"control-plane=controller-manager": 1},
+            ]
+        except Exception as e:
+            logger.error(f"Error during ODF client CSV's installation: {e}")
+            return False
+
+        try:
+            pods_are_running = wait_for_pods_to_be_in_statuses_concurrently(
+                app_selectors_to_resource_count_list,
+                self.namespace_client,
+                timeout_wait_pod * 60,
+            )
+        except Exception as e:
+            logger.error(f"Error during ODF client pods status check: {e}")
+            pods_are_running = False
+
+        if not pods_are_running:
+            logger.error(
+                f"ODF client pods with labels {app_selectors_to_resource_count_list} are not running"
+            )
+            return False
+        else:
+            logger.info("ODF client pods are running, CSV's are installed")
+            return True
+
+    @kubeconfig_exists_decorator
+    def storage_client_exists(self):
+        """
+        Check if the storage client exists
+
+        Returns:
+            bool: True if storage client exists, False otherwise
+        """
+        ocp = OCP(
+            kind=constants.STORAGECLIENTS,
+            namespace=self.namespace_client,
+            cluster_kubeconfig=self.cluster_kubeconfig,
+        )
+        return ocp.check_resource_existence(
+            timeout=self.timeout_check_resources_exist,
+            resource_name=constants.STORAGE_CLIENT_NAME,
```

Comment:
This should be taken from config. There is already a value for it in default config (https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L145).

Reply:
corrected, thanks